tox.ini

```diff
@@ -10,8 +10,9 @@ isolated_build = true
 
 [testenv]
 setenv =
+    conda: CONDA_BUILD_SYSROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
```

Comment:
Might need to add this to the templates, it's an ugly hack but seems to be required on Azure...

Reply:
Fixed in the template!